mlflow/server/auth/__init__.py

```diff
@@ -8,15 +8,71 @@
 """
 
 import logging
+import uuid
```

Comment:
Importing this module mutates the app, which caused a few tests to fail. Can we separate test?

Reply:
Adding `tests/server/auth` here might work:

https://github.com/mlflow/mlflow/blob/4728afe1a6b77a505653de19ee73dc7869ce889f/conftest.py#L45-L50

The tested in this list are opt-in tests that are ignored by default.

Reply:
i see, added!